internal/pkg/skuba/kubeadm/images.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/SUSE/skuba/pkg/skuba"
 )
 
-func SetContainerImagesWithClusterVersion(initConfiguration *kubeadmapi.InitConfiguration, clusterVersion *version.Version) {
+func setContainerImagesWithClusterVersion(initConfiguration *kubeadmapi.InitConfiguration, clusterVersion *version.Version) error {
```

Comment:
also do we need to return an `error` here?

Reply:
Thanks! Addressed.